tests/e2e/scenarios/harvest_nucl_th_and_jlab_curation/ArxivService/interaction_0.yaml

```diff
@@ -0,0 +1,48 @@
+callbacks: []
```

Comment:
you can try to reuse the ones we have for the other arxiv harvests, reusing the same file will use a lot less space (and potentially speed up the tests, small amount of authors and such).

Reply:
Are you refering to the whole arXiv harvest? If yes, I can't since this is the `nucl-th` record that we have to test in particular.

Reply:
good idea, actually the size of the cassettes I was using is quite bigger than the others :+1:

Reply:
done, it's quicker ;)